src/git.ts

```diff
@@ -314,8 +314,9 @@ export async function deploy(action: ActionInterface): Promise<Status> {
 
         // If the push failed for any fatal reason other than being rejected,
         // there is a problem
-        if (!rejected && pushResult.stderr.trim().startsWith('fatal:'))
+        if (!rejected && pushResult.stderr.split(/\n/).some(s => s.trim().startsWith('fatal:')) {
```

Comment:
This line is flagging a syntax error: 

```
/home/runner/work/github-pages-deploy-action/github-pages-deploy-action/src/git.ts
  317:96  error  Parsing error: ')' expected
```

Reply:
That's my bad! Thought the change was too trivial to test but Murphy's law is quite right. Fixed now.